### ISSUE TITLE
Need the code for evaluation of the pipeline model

### ISSUE THREAD

**Issue by neverneverendup** [jacklxc/ParagraphJointModel#3]
Dear author, your research work is excellent, thank you for your excellent work!
I would like to ask if it is convenient for you to provide the code for the evaluation of the pipeline model containing your rationale selection module and label prediction module using compact paragraph encoding. There seems to be no relevant code in the current git repository.
Thanks!

**Comment by jacklxc** [comment]
Thank you for your appreciation. Some file contains the following imports, which are for evaluations.
```
from lib.data import GoldDataset, PredictedDataset
from lib import metrics
```
These evaluation codes are directly taken from [SciFact's repository](https://github.com/allenai/scifact).

**Comment by neverneverendup** [comment]
Dear author,
Have the 2 component (rationale selection model and label prediction model) of your pipeline model pretrained on the FEVER dataset?  There seems no code for pretrained the rationale selection model with compact paragraph embedding on FEVER dataset~

**Comment by jacklxc** [comment]
Please refer to FEVER Pre-training section in README.
